File: bcw_2023summer_keepr/Models/Vault.cs
namespace bcw_2023summer_keepr.Models
{
    public class Vault : RepoItem<int>
    {
        public string CreatorId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Img { get; set; }
        public bool? isPrivate { get; set; }
        public Profile Creator { get; set; }
    }
}

File: bcw_2023summer_keepr.client/src/models/Vault.js
export class Vault {
	constructor(data) {
		this.id = data.id,
			this.creatorId = data.creatorId,
			this.name = data.name,
			this.description = data.description,
			this.img = data.img,
			this.isPrivate = data.isPrivate,
			this.creator = data.creator
	}
}